components/licence-sign-in.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function LicenceSignIn ({
+	displayName = null,
+	url = null,
```

Comment:
```suggestion
	url = '',
```
However: is it ok to not have a url?  
Shall we have a fallback value? I wonder if this should be required?